### PR TITLE
Fix grazing angle update issue

### PR DESCRIPTION
**Pull request by AbbyGi**

### CODE REVIEW

**Comment by AbbyGi** [comment]
I plan on working on this more possibly later in the week if I have time

File: sirepo_bluesky/sirepo_detector.py [hunk 1]
```diff
@@ -107,6 +108,9 @@ def trigger(self):
 
         if not self.source_simulation:
             if self.sirepo_component is not None:
+                for i in self.data['models']['beamline']:
```
Comment (mrakitin) [inline sirepo_bluesky/sirepo_detector.py:111]: Maybe rename `i` to something more explanatory?

File: sirepo_bluesky/sirepo_detector.py [hunk 2]
```diff
@@ -116,6 +120,16 @@ def trigger(self):
                                                    'title',
                                                    self.parents['par' + str(i)])
                     element[real_field] = x
+                    if self.parents[f'par{i}'] in self.autocompute_params.keys() and 'grazingAngle' in dict_key:
+                        grazing_vecs_dict = {}
+                        autocompute_key = f'{self.parents[f"par{i}"]}_sirepo_autocomputeVectors'
```
Comment (mrakitin) [inline sirepo_bluesky/sirepo_detector.py:125]: Wow, f-string within another f-string, never saw it before, but I hope it works as expected!

File: sirepo_bluesky/sirepo_flyer.py [hunk 3]
```diff
@@ -184,6 +184,19 @@ def kickoff(self):
         data, schema = sb.auth(self.sim_code, self.sim_id)
         self._copies = []
         self._srw_files = []
+        autocompute_data = {}
+        # grazing angle; check params_to_change
+        for i in data['models']['beamline']:
```
Comment (mrakitin) [inline sirepo_bluesky/sirepo_flyer.py:189]: Same suggestion here.

**Pull request merged**